app/components/QuickTemplate.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React, { Component } from 'react';
+import { Sidebar, Button, Form, Header, Icon, Grid } from 'semantic-ui-react';
+
+import styles from './QuickTemplate.css';
+
+const fs = require('fs');
+
+export default class QuickTemplate extends Component {
+  props: {
+    dispatchButtonClick: (action: string) => void,
+    openTemplate: (deployPath: string) => void,
+    isQuickTemplateOpen: boolean
+  }
+
+  generateGrid() {
+    const templates = JSON.parse(fs.readFileSync('./app/template/output.json', 'utf8'));
```

Comment:
This parsing has to be done each time this function is called(so each render of this component will cause this). This could be improved for sure by caching data.